cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ForLoopStatement.java

```diff
@@ -9,31 +9,16 @@
  *******************************************************************************/
 package io.cloudslang.lang.entities;
 
-import org.apache.commons.lang3.Validate;
-
 import java.io.Serializable;
 
-public abstract class ForLoopStatement implements Serializable{
-
-    private static final long serialVersionUID = -1467375536631244911L;
-
-    private final String collectionExpression;
-
-    public ForLoopStatement(String collectionExpression) {
-        Validate.notBlank(collectionExpression, "for loop collection expression cannot be empty");
+public abstract class ForLoopStatement extends LoopStatement implements Serializable{
 
-        this.collectionExpression = collectionExpression;
+    public ForLoopStatement(String expression) {
```

Comment:
it was there just for the hierarchy.. so the new hierarchy will be:

LoopStatement
    ListForLoopStatement
    MapForLoopStatement
    AsyncLoopStatement